cartoframes/data/observatory/repository/category_repo.py

```diff
@@ -0,0 +1,41 @@
+from cartoframes.data.observatory.repository.repo_client import RepoClient
+
+
+def get_category_repo():
+    return CategoryRepository()
+
+
+class CategoryRepository(object):
+
+    def __init__(self):
+        self.client = RepoClient()
+
+    def get_all(self):
+        return self._to_categories(self.client.get_categories())
+
+    def get_by_id(self, category_id):
+        result = self.client.get_categories('id', category_id)
+
+        if len(result) == 0:
+            return None
+
+        return self._to_category(result[0])
+
+    def get_by_country(self, iso_code):
```

Comment:
is this `iso_code3`?

Reply:
Yes, I had an internal fight about using iso_code vs iso_code3 and missed this one xD. 

Anyway I saw this one is not actually possible since the categories have not relation with country in the DB model. Or do we intend to get the datasets of that country and for each one get their categories and put them together?

Reply:
IMO this method is not needed in the discovery API